python/python-video-generation/code/Pipfile

```diff
@@ -0,0 +1,17 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+flake8 = "*"
+pylint = "*"
+autopep8 = "*"
+
+[packages]
+moviepy = "==1.0.3"
+gizeh = "==0.1.11"
```

Comment:
@anddriex `Gizeh` is still part of the dependencies. Let's remove it since it is not being used anymore.